eval/Containerfile

```diff
@@ -1,18 +1,20 @@
 FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
 
 RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++ \
+    && dnf install -y libcudnn8 libcudnn8-devel \
```

Comment:
can't this be part of the previous `dnf` call?

Reply:
I'm currently trying a 100 different things, bc the build is failing - you don't need to review this atm 😃

Reply:
hence the `WIP` but if you want to try building this image locally to see what the Containerfile is missing, I'd appreciate that!

Reply:
done